polaris-react/package.json

```diff
@@ -120,6 +120,7 @@
     "last 3 safari versions",
     "last 3 chromeandroid versions",
     "last 1 firefoxandroid versions",
-    "ios >= 13.4"
+    "ios >= 13.4",
+    "node 20.10.0"
```

Comment:
Why is this fixed to 20.10.0? Shouldn't it be `>= 20.10.0`? I am on `20.16.0` and this is causing build errors